specs/serving/knative-api-specification-1.0.md

```diff
@@ -2215,6 +2215,26 @@ Max: 1
    <td><strong>Schema Requirement</strong>
    </td>
   </tr>
+  <tr>
+   <td><code>readOnlyRootFilesystem</code>
+   </td>
+   <td>boolean
+   </td>
+   <td>As specified in Kubernetes <a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#securitycontext-v1-core">core/v1.SecurityContext</a>
+   </td>
+   <td>RECOMMENDED, if securityContext is supported.
+   </td>
+  </tr>
+  <tr>
+   <td><code>runAsGroup</code>
+   </td>
+   <td>int
+   </td>
+   <td>As specified in Kubernetes <a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#securitycontext-v1-core">core/v1.SecurityContext</a>
+   </td>
+   <td>REQUIRED, if securityContext is supported.
+   </td>
+  </tr>
```

Comment:
Ditto, in terms of [PodSecurityStandards](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted).

We probably also need to update the [Runtime contract](https://github.com/knative/specs/blob/main/specs/serving/runtime-contract.md#user).